redirects.yaml

```diff
@@ -83,6 +83,7 @@ tv/meet/christian-barkowsky: https://youtu.be/S9GeQXy4jTo
 tv/meet/moritz-vondano: https://youtu.be/xEC4h8Q0reY
 tv/meet/andreas-schempp: https://youtu.be/x16YkivYaOY
 tv/meet/ingolf-steinhardt: https://youtu.be/_pYywkVxUUo
+tv/christian-feneberg: https://youtu.be/juimu_GKUAI
```

Comment:
Should this not be `tv/meet/christian-feneberg`?

Reply:
Oh right, I missed this. Do you want to update that @may17 ?

Reply:
Should be fixed with https://github.com/contao/to.contao.org/pull/37 already